readthedocs/projects/forms.py

```diff
@@ -357,6 +358,17 @@ def clean_conf_py_file(self):
             )  # yapf: disable
         return filename
 
+    def clean_rtd_conf_file(self):
+        filename = self.cleaned_data.get("rtd_conf_file", "").strip()
+        if filename and ".readthedocs.yml" not in filename:
```

Comment:
I think we allow multiple variations of this file, so not sure we need strict filename validation...

The bigger benefit would be checking if the file exists, but that's harder :)

Reply:
I don't think we need to require any convention in the filename nor path.

Currently, we don't fail a build in the absence of a config file, so I guess that we should start doing that - _if_ a custom config file is defined. Otherwise, people might add a wrong path and the build runs and you spend 1 hour looking the wrong place :)